File: C#-Ödev 10/C#-Ödev 10/Form1.cs
namespace C__Ödev_10
{
	public partial class Form1 : Form
	{
		public Form1()
		{
			InitializeComponent();
		}

		private void button1_Click(object sender, EventArgs e)
		{
			int[] sayilar = { 15, 9, 35, 75, 5, 3, 20, 95, 11, 10 };
			for (int i = 0; i < sayilar.Length; i++)
			{
				if (sayilar[i] % 2 == 0)
				{
					listBox1.Items.Add(sayilar[i]);
				}
			}
		}
	}
}

File: WinFormsApp6/WinFormsApp6/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs)
        ListBox1.Items.Add("Rota:" + ComboBox1.Text + " - " + ComboBox2.Text + " Tarih: " + DateTimePicker1.Text + " Saat: " + MaskedTextBox1.Text + " Yolcu Bilgileri~Adı Soyadı:" + TextBox1.Text + "TC:" + MaskedTextBox2.Text + "Telefon:" + MaskedTextBox3.Text)
        MessageBox.Show("Yolcu Kaydı Yapıldı.")
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Label9.Text = ComboBox2.Text
        ComboBox2.Text = ComboBox1.Text
        ComboBox1.Text = Label9.Text
    End Sub
End Class
